components/ClassInviteTable.js

```diff
@@ -52,11 +55,10 @@ export default function ClassInviteTable({
           body: JSONdata
         });
         if (res.status === 403) {
```

Comment:
In the event that the delete operation fails, the browser will falsely indicate that the operation was successful.  The code does catch one failure case (403 error), but there might be other error codes not accounted for (such as a 500 error). The ideal solution I believe is to change the if statement to be the following: 

if (res.status !== 200) {
  // handle error 
}

I think it's ok to merge your PR anyway, but we should log this edge case for a future issue.